src/liquid.py

```diff
@@ -338,10 +363,25 @@ def npt_simulation(self, temperature, pressure, simnum):
                 logger.info("You may tail -f %s/npt.out in another terminal window\n" % os.getcwd())
                 _exec(cmdstr, copy_stderr=True, outfnm='npt.out')
             else:
-                queue_up(wq, command = cmdstr+' &> npt.out',
+                queue_up(wq, command = cmdstr+' > npt.out 2>&1 ',
```

Comment:
The old command will crash on Ubuntu 16.04 with newer bash shells. We fixed it by using the new command.